rotary/utils.py

```diff
@@ -100,3 +100,34 @@ def symlink_or_compress(in_file_path, out_file_path):
         os.symlink(in_file_path, out_file_path)
     else:
         gzip_file(in_file_path, out_file_path)
+
+
+def get_contamination_reference_files(config_entry: list, db_path: str, phix_accession: str, human_accession: str):
```

Comment:
Change the name of this function to `get_contamination_reference_file_paths.` Generally, when I put the word `file` of a function, I expect it to return a file handle, which is the object created by the `open('file/path).` This function returns the paths to the files, not the files or handles themselves.

Reply:
Done - good point!